humility-arch-cortex/src/scs.rs

```diff
@@ -71,7 +71,7 @@ register_offs!(SWTF_CTRL, 0x0,
     pub es0, set_es0: 0;
 );
 
-#[derive(Copy, Clone, Debug, PartialEq)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
```

Comment:
What lint recommended adding this extra trait?

Reply:
[derive_partial_eq_without_eq](https://rust-lang.github.io/rust-clippy/master/index.html#derive_partial_eq_without_eq)

Reply:
Huh, that's a strange lint.  Basically says to derive the trait because you can.